### PR TITLE
Fixing install screen button issue

### PR DESCRIPTION
**Pull request by dcluna**
Non-stretched buttons in the install screens.

### CODE REVIEW

File: app/src/org/commcare/android/view/SquareButton.java [hunk 1]
```diff
@@ -27,7 +27,12 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
...
         int h = MeasureSpec.getSize(heightMeasureSpec);
 
-        w = Math.max(w, h);
+        if(w == 0 || h == 0) { // this can happen in the home screen gridView, and then we'll have an invisible view
+            w = Math.max(w, h);
+        } else {
+            w = Math.min(w, h);
+        }
+
         h = w;
```
Comment (phillipm) [inline app/src/org/commcare/android/view/SquareButton.java]: This assignment seems useless

File: app/src/org/commcare/android/view/SquareButton.java [hunk 2]
```diff
@@ -27,7 +27,12 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
         int w = MeasureSpec.getSize(widthMeasureSpec);
         int h = MeasureSpec.getSize(heightMeasureSpec);
```
Comment (phillipm) [inline app/src/org/commcare/android/view/SquareButton.java]: Are these the canvas size or the button size?

File: app/src/org/commcare/android/view/SquareButton.java [hunk 3]
```diff
@@ -27,7 +27,12 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
...
 
-        w = Math.max(w, h);
+        if(w == 0 || h == 0) { // this can happen in the home screen gridView, and then we'll have an invisible view
```
Comment (phillipm) [inline app/src/org/commcare/android/view/SquareButton.java]: I'm sort of irrationally picky about this, but can you put comments on their own lines, and inside of the conditional body that they correspond to?

File: app/src/org/commcare/android/view/SquareButton.java (hunk 1 above)
Comment (dcluna) [inline app/src/org/commcare/android/view/SquareButton.java]: It is

File: app/src/org/commcare/android/view/SquareButton.java (hunk 2 above)
Comment (phillipm) [inline app/src/org/commcare/android/view/SquareButton.java]: You can avoid the comment by naming 'w' and 'h' to 'canvasWidth' and 'canvasHeight'

Comment (dcluna) [inline app/src/org/commcare/android/view/SquareButton.java]: Button size

File: app/src/org/commcare/android/view/SquareButton.java (hunk 3 above)
Comment (phillipm) [inline app/src/org/commcare/android/view/SquareButton.java]: Please put a space in between 'if' and the paren. 'if' is not a function, but rather a keyword, so we shouldn't be using functional call syntax on it.

Comment (phillipm) [inline app/src/org/commcare/android/view/SquareButton.java]: This comment is pretty confusing. When do we have an invisible view? Are we avoiding an invisible view?

**Pull request merged**